packages/jellyfish-transaction/src/script/mapping.ts

```diff
@@ -414,6 +424,7 @@ export const OP_CODES = {
       data: icxClaimDFCHTLC
     })
   },
+
```

Comment:
extra line?

Reply:
I reckon there was this extra line before we started merging the ICX DfTx